draft-ietf-webtrans-http3.md

```diff
@@ -546,6 +548,23 @@ Specification:
 
 : This document.
 
+Name:
+
+: H3_WEBTRANSPORT_SESSION_GONE
+
+Value:
+
+: 0x170d7b68
+
+Description:
+
+: WebTransport data stream reset because the associated WebTransport session
```

Comment:
since the text applies to both read and write, we can be more generic in the description

```suggestion
: WebTransport data stream aborted because the associated WebTransport session
```